test/functional/api/buffer_spec.lua

```diff
@@ -249,6 +251,17 @@ describe('api/buf', function()
       eq(1, funcs.exists('b:lua'))
       curbufmeths.del_var('lua')
       eq(0, funcs.exists('b:lua'))
+      curbuf('set_var', 'changedtick', true)
```

Comment:
`set_var` sets values in `b:` dictionary, `let b:changedtick` uses a unique (!) workaround that prevents it from working, use `let b:.changedtick`.